src/main/java/wooteco/subway/reopository/StationRepository.java

```diff
@@ -0,0 +1,43 @@
+package wooteco.subway.reopository;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import org.springframework.stereotype.Repository;
+import wooteco.subway.domain.Station;
+import wooteco.subway.exception.NotFoundException;
+import wooteco.subway.reopository.dao.StationDao;
+import wooteco.subway.reopository.entity.StationEntity;
+
+@Repository
+public class StationRepository {
+
+    private final StationDao stationDao;
+
+    public StationRepository(StationDao stationDao) {
+        this.stationDao = stationDao;
+    }
+
+    public Long save(Station station) {
+        return stationDao.save(new StationEntity(station.getName()));
+    }
+
+    public Station findById(Long id, String errorMessage) {
+        StationEntity stationEntity = stationDao.findById(id)
+                .orElseThrow(() -> new NotFoundException(errorMessage));
+        return new Station(stationEntity.getId(), stationEntity.getName());
+    }
+
+    public List<Station> findAll() {
+        List<StationEntity> list = stationDao.findAll();
+        return list.stream().map(entity -> new Station(entity.getId(), entity.getName())).collect(toList());
```

Comment:
```suggestion
        return list.stream()
                .map(entity -> new Station(entity.getId(), entity.getName()))
                .collect(toList());
```
가독성을 위해 각 체이닝마다 개행을 넣어주면 어떨까 제안드립니다~

추가로 IntelliJ에서는 `show method chain hints` 기능으로 메서드 체이닝마다 반환되는 타입을 보여주는 기능도 있는데요. 이 기능을 활용하기 위해서도 개행이 필요하네요.

<img width="869" alt="스크린샷 2022-05-20 오후 5 46 59" src="https://user-images.githubusercontent.com/30178507/169491389-abb19ad7-51cb-4601-8c8b-0a5edd4549e2.png">

Reply:
철시 ~ ㅎㅎ 
실수로 개행을 맞추지 못했는데, 그 덕분에 `show method chain hints` 기능의 명칭을 알게 되어서 기쁘네요 👍🏻 
기술적인 것과 더불어 IDE 의 단축키나 기능을 활용하는 것도 효율을 올리는 방법중 하나인 것 같아요. 
그래서 요새 의식적으로 단축키와 친해지려고 하고 있습니다. 요 기능도 재밌게 사용해보겠습니다. 감사합니다. 피드백 내용은 반영하겠습니다 !!! 내용 너무 좋네요